From 69132ef4591cfde578f52dffc91c678b9cc52caa Mon Sep 17 00:00:00 2001
From: Ismo Puustinen <ismo.puustinen@intel.com>
Date: Tue, 8 Nov 2016 13:14:45 +0200
Subject: [PATCH 4/4] Update to libefivar 27.

---
 src/libostree/ostree-bootloader-efiblob.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/libostree/ostree-bootloader-efiblob.c b/src/libostree/ostree-bootloader-efiblob.c
index f5f0fab..68d6f39 100644
--- a/src/libostree/ostree-bootloader-efiblob.c
+++ b/src/libostree/ostree-bootloader-efiblob.c
@@ -153,10 +153,10 @@ _ostree_bootloader_efiblob_write_config (OstreeBootloader          *bootloader,
       efi_load_option *opt = (efi_load_option *) buf;
       /* printf("description (%u): '%s'\n", num, efi_loadopt_desc(opt)); */
 
-      if (strcmp((const char *) efi_loadopt_desc(opt), target_label) == 0) {
+      if (strcmp((const char *) efi_loadopt_desc(opt, bufsize), target_label) == 0) {
         /* boot next to the found target */
         r = efi_set_variable(EFI_GLOBAL_GUID, "BootNext", (uint8_t *) &num,
-            sizeof(num), attributes);
+            sizeof(num), attributes, 0644);
         if (r != 0) {
           /* we can't set the boot target for some reason, this is bad */
           /* printf("error setting variable: %s\n", strerror(errno)); */
-- 
2.7.4

